### PR TITLE
Swedish-translation-20151119

### PR DESCRIPTION
**Pull request by sebastianlorant**

### CODE REVIEW

**Comment by MilosKozak** [comment]
there is syntax error here. https://github.com/nightscout/cgm-remote-monitor/pull/1365/files#diff-29e70f5c52b22198247a7936ea3f1d7dR4938
fix it before merge pls

File: lib/language.js [hunk 1]
```diff
@@ -4811,138 +4858,161 @@ function init() {
...
+      ,sv: 'Beskriver under hur lång tid insulinet verkar' Varierar mellan patienter. Typisk varaktighet 3-4 timmar.'
       ,nb: 'Representerer typisk insulinvarighet. Varierer per pasient og per insulin type. Vanligvis 3-4 timer for de fleste typer insulin og de fleste pasientene. Noen ganger også kalt insulinlevetid.'
       ,fi: 'Kertoo insuliinin tyypillisen vaikutusajan. Vaihtelee potilaan ja insuliinin tyypin mukaan. Tyypillisesti 3-4 tuntia pumpuissa käytettävällä insuliinilla.'
       }
     ,'Insulin to carb ratio (I:C)' : {
       cs: 'Inzulíno-sacharidový poměr (I:C).'
       ,ro: 'Rată insulină la carbohidrați (ICR)'
+      ,sv: 'Insulin-Kolhydratskvot'
       ,nb: 'IKH forhold'
       ,fi: 'Insuliiniannoksen hiilihydraattisuhde (I:HH)'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4947)

File: lib/language.js [hunk 2]
```diff
@@ -4811,138 +4858,161 @@ function init() {
...
       }
     ,'hours' : {
       cs: 'hodiny'
       ,ro: 'ore'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4951)

File: lib/language.js [hunk 3]
```diff
@@ -4811,138 +4858,161 @@ function init() {
...
+      ,sv: 'timmar'
       ,nb: 'timer'
       ,fi: 'tuntia'
       }
     ,'g/hour' : {
       cs: 'g/hod'
       ,ro: 'g/oră'
+      ,sv: 'g/timme'
       ,nb: 'g/t'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4960)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4960)

File: lib/language.js [hunk 4]
```diff
@@ -4811,138 +4858,161 @@ function init() {
...
       ,ro: 'Durata de acțiune a insulinei'
+      ,sv: 'verkningstid för insulin (DIA)'
       ,nb: 'Insulin varighet'
       ,fi: 'Insuliinin vaikutusaika (DIA)'
       }
     ,'Represents the typical duration over which insulin takes effect.  Varies per patient and per insulin type. Typically 3-4 hours for most pumped insulin and most patients. Sometimes also called insulin lifetime.' : {
       cs: 'Představuje typickou dobu, po kterou inzulín působí. Bývá různá podle pacienta a inzulínu. Typicky 3-4 hodiny pro pacienty s pumpou.'
       ,ro: 'Reprezintă durata tipică pentru care insulina are efect. Este diferită la fiecare pacient și pentru fiecare tip de insulină'
+      ,sv: 'Beskriver under hur lång tid insulinet verkar' Varierar mellan patienter. Typisk varaktighet 3-4 timmar.'
       ,nb: 'Representerer typisk insulinvarighet. Varierer per pasient og per insulin type. Vanligvis 3-4 timer for de fleste typer insulin og de fleste pasientene. Noen ganger også kalt insulinlevetid.'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected '{' and instead saw 'de'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4939)

File: lib/language.js (hunk 1 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4947)

File: lib/language.js [hunk 5]
```diff
@@ -4811,138 +4858,161 @@ function init() {
...
       }
     ,'hours' : {
       cs: 'hodiny'
       ,ro: 'ore'
+      ,sv: 'timmar'
       ,nb: 'timer'
       ,fi: 'tuntia'
       }
     ,'g/hour' : {
       cs: 'g/hod'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4957)

File: lib/language.js (hunk 2 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4951)

File: lib/language.js [hunk 6]
```diff
@@ -4811,138 +4858,161 @@ function init() {
...
       }
     ,'Represents the typical duration over which insulin takes effect.  Varies per patient and per insulin type. Typically 3-4 hours for most pumped insulin and most patients. Sometimes also called insulin lifetime.' : {
       cs: 'Představuje typickou dobu, po kterou inzulín působí. Bývá různá podle pacienta a inzulínu. Typicky 3-4 hodiny pro pacienty s pumpou.'
       ,ro: 'Reprezintă durata tipică pentru care insulina are efect. Este diferită la fiecare pacient și pentru fiecare tip de insulină'
+      ,sv: 'Beskriver under hur lång tid insulinet verkar' Varierar mellan patienter. Typisk varaktighet 3-4 timmar.'
       ,nb: 'Representerer typisk insulinvarighet. Varierer per pasient og per insulin type. Vanligvis 3-4 timer for de fleste typer insulin og de fleste pasientene. Noen ganger også kalt insulinlevetid.'
       ,fi: 'Kertoo insuliinin tyypillisen vaikutusajan. Vaihtelee potilaan ja insuliinin tyypin mukaan. Tyypillisesti 3-4 tuntia pumpuissa käytettävällä insuliinilla.'
       }
     ,'Insulin to carb ratio (I:C)' : {
       cs: 'Inzulíno-sacharidový poměr (I:C).'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4943)

File: lib/language.js (hunk 3 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4960)

File: lib/language.js [hunk 7]
```diff
@@ -4811,138 +4858,161 @@ function init() {
...
       cs: 'Doba působnosti inzulínu (DIA)'
       ,ro: 'Durata de acțiune a insulinei'
+      ,sv: 'verkningstid för insulin (DIA)'
       ,nb: 'Insulin varighet'
       ,fi: 'Insuliinin vaikutusaika (DIA)'
       }
     ,'Represents the typical duration over which insulin takes effect.  Varies per patient and per insulin type. Typically 3-4 hours for most pumped insulin and most patients. Sometimes also called insulin lifetime.' : {
       cs: 'Představuje typickou dobu, po kterou inzulín působí. Bývá různá podle pacienta a inzulínu. Typicky 3-4 hodiny pro pacienty s pumpou.'
       ,ro: 'Reprezintă durata tipică pentru care insulina are efect. Este diferită la fiecare pacient și pentru fiecare tip de insulină'
+      ,sv: 'Beskriver under hur lång tid insulinet verkar' Varierar mellan patienter. Typisk varaktighet 3-4 timmar.'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4938)

File: lib/language.js (hunk 3 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4960)

File: lib/language.js [hunk 8]
```diff
@@ -4811,138 +4858,161 @@ function init() {
...
       ,nb: 'Representerer typisk insulinvarighet. Varierer per pasient og per insulin type. Vanligvis 3-4 timer for de fleste typer insulin og de fleste pasientene. Noen ganger også kalt insulinlevetid.'
       ,fi: 'Kertoo insuliinin tyypillisen vaikutusajan. Vaihtelee potilaan ja insuliinin tyypin mukaan. Tyypillisesti 3-4 tuntia pumpuissa käytettävällä insuliinilla.'
       }
     ,'Insulin to carb ratio (I:C)' : {
       cs: 'Inzulíno-sacharidový poměr (I:C).'
       ,ro: 'Rată insulină la carbohidrați (ICR)'
+      ,sv: 'Insulin-Kolhydratskvot'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4945)

File: lib/language.js (hunk 4 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected '{' and instead saw 'de'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4939)

File: lib/language.js [hunk 9]
```diff
@@ -4811,138 +4858,161 @@ function init() {
...
       ,nb: 'IKH forhold'
       ,fi: 'Insuliiniannoksen hiilihydraattisuhde (I:HH)'
       }
     ,'hours' : {
       cs: 'hodiny'
       ,ro: 'ore'
+      ,sv: 'timmar'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4952)

File: lib/language.js (hunk 5 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4957)

File: lib/language.js (hunk 6 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4943)

File: lib/language.js (hunk 7 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4938)

File: lib/language.js (hunk 8 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4945)

File: lib/language.js (hunk 9 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1367486668/issues/source?bid=2452379&fileBranchId=2976078#l4952)

**Comment by sebastianlorant** [comment]
Ok,  so you fixed it? :)

Have a nice weekend!
Den 20 nov 2015 10:09 skrev "Milos Kozak" <notifications@github.com>:

**Pull request merged**